### DISCUSSION TITLE
nodejs Headless CMS support in bun or new CMS

### DISCUSSION THREAD

**Discussion by elycheikhsmail** [oven-sh/bun#5553] [General]
is Nodejs Headless CMS like Strapi are yet supported in bun ? is there there a need for news Headless CMS written for "scratch" to play nice in bun ecosystem ?

**Comment by jonasb** [comment]
I'm working on a headless CMS, [Dossier](https://www.dossierhq.dev/), where I support Bun (in addition to Node, Deno and browser).

At least so far I'm using different database library for each runtime (and database engine since Dossier supports SQLite and Postgres). So for Bun I'm using `bun:sqlite`. Other than that, Bun has been quite easy to support, the compatibility has been good.

If you're interested, take a look at the source for [bun-sqlite](https://github.com/dossierhq/dossierhq/tree/main/libraries/bun-sqlite).

The bun-sqlite library is currently not published to npm, but just shout if you're interested in giving it a go and I can publish it.

**Reply by Frikadellios** [reply]
Hello, 

Awesome project, you can use typesafe-i18n with Inlang for auto translation for all locales. 

And will be great use tiptap vue for better performance (draws and other things) 

I am using :
- Primo CMS
- Tina CMS
- Vrite CMS
- Appwrite and Xata for live updating without rebuilds..

That's was hard.. use Astrojs for Ukrainian projects)

But, I am noob and learning alone, so that's to hard for me) 

Try to explore vike.dev - new vite ) 

And nuejs..

That's a new tools 🔥🔥
Have a good mood) 😁

**Comment by elycheikhsmail** [comment]
@jonasb sqlite is good for testing in local but not goos in production in most cases, sereval reading is possible at the same time but only one writing at time.
